Automattic/jetpack
# [Carousel block]: Fix comments indicator placement on Twenty Twenty

Fixes 318-gh-Automattic/view-design

#### Changes proposed in this Pull Request:

* Explicitly sets `display: inline-block` on the carousel icons to prevent CSS issues in themes that set `display: block` on svgs.

| Before | After |
| --- | --- |
| <img width="317" alt="Screen Shot 2021-07-01 at 1 48 36 PM" src="https://user-images.githubusercontent.com/63313398/124187724-1696ae00-da73-11eb-8ed8-60f05865d4b6.png"> | <img width="321" alt="Screen Shot 2021-07-01 at 1 48 27 PM" src="https://user-images.githubusercontent.com/63313398/124187749-1d252580-da73-11eb-998a-49b14dcb065e.png"> |


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Add a gallery with at least one image to a post. Add a couple comments to one of the images.
* Set the site's theme to Twenty Twenty
* Open the image in the carousel and verify that the comments indicator looks correct
